qa/rpc-tests/test_framework/blocktools.py

```diff
@@ -59,6 +59,12 @@ def create_coinbase(height, pubkey = None):
     else:
         coinbaseoutput.scriptPubKey = CScript([OP_TRUE])
     coinbase.vout = [ coinbaseoutput ]
+
+    # Make sure the coinbase is at least 100 bytes
```

Comment:
doens't make much of a difference but why not making the below change conditional to activation time param?